subprojects/performance/src/templates/root-project/gradle.properties

```diff
@@ -0,0 +1,2 @@
+<% if (binding.hasVariable("daemonMemory")) { %>org.gradle.jvmargs=-Xmxs${binding.getVariable("daemonMemory")} -Xmx${binding.getVariable("daemonMemory")}<% } %>
```

Comment:
We should probably add `compilerMemory` and `testRunnerMemory` here instead of inline in the build script, so they are easier to see/change at a glance.